mail/mail-policy.go

```diff
@@ -318,3 +318,48 @@ func SendMailRenewDraft(policy models.Policy, from, to, cc Address, flowName str
 		IsApp:       true,
 	})
 }
+
+func SendMailRenewDraftV2(policy models.Policy, from, to, cc Address, flowName string, hasMandate bool) error {
```

Comment:
if I'm not mistaken, policy should have a `HasMandate` flag now so it shouldn't be necessary to pass it as an extra argument